docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3"
```

Comment:
I don't think we need this docker-compose file now, since we'll run it from the top-level dev env?